3p/bodymovinanimation.js

```diff
@@ -20,6 +20,15 @@ import {loadScript} from './3p';
 import {parseJson} from '../src/json';
 import {setStyles} from '../src/style';
 
+const libSourceUrl = {
+  'canvas':
+    'https://cdnjs.cloudflare.com/ajax/libs/lottie-web/5.7.6/lottie_canvas.min.js',
```

Comment:
Bodymovin was renamed to Lottie, as mentioned in the original issue, hence the change in URLs.

See https://github.com/airbnb/lottie-web/blob/master/History.md#v-500 and https://airbnb.io/lottie/#/.

Reply:
Not required for this pull request: would it be valuable to (in addition) alias this component as `amp-lottie-animation`?